### PR TITLE
creeps stacked/party data

### LINKED ISSUE #1147: Party data
like [Dotabuff does](https://www.reddit.com/r/DotA2/comments/4vpfi0/dotabuff_now_shows_who_is_partied_with_who/)

### PR DESCRIPTION
**Pull request by howardchung**
fixes #1147 

* Adds a new table to store GC match data (replay_salt, match_id, cluster, parties, series_id, series_type)
* Adds new extracted entity properties (creeps stacked, camps stacked, rune pickups, obs placed, sen placed)

TODO:
- [ ] delete old redis key replay_url
- [x] run schema changes on prod

cassandra:
```
ALTER TABLE player_matches ADD obs_placed int;
ALTER TABLE player_matches ADD sen_placed int;
ALTER TABLE player_matches ADD creeps_stacked int;
ALTER TABLE player_matches ADD camps_stacked int;
ALTER TABLE player_matches ADD rune_pickups int;
ALTER TABLE player_matches ADD obs_left_log text;
ALTER TABLE player_matches ADD sen_left_log text;
```
postgres:
```
CREATE TABLE match_gcdata(
match_id bigint PRIMARY KEY,
 cluster int,
replay_salt int,
series_id int,
series_type int,
parties json
);
ALTER TABLE player_matches ADD obs_placed int;
ALTER TABLE player_matches ADD sen_placed int;
ALTER TABLE player_matches ADD creeps_stacked int;
ALTER TABLE player_matches ADD camps_stacked int;
ALTER TABLE player_matches ADD rune_pickups int;
ALTER TABLE player_matches ADD obs_left_log json[];
ALTER TABLE player_matches ADD sen_left_log json[];
ALTER TABLE match_logs ADD obs_placed int;
ALTER TABLE match_logs ADD sen_placed int;
ALTER TABLE match_logs ADD creeps_stacked int;
ALTER TABLE match_logs ADD camps_stacked int;
ALTER TABLE match_logs ADD rune_pickups int;
```

### CODE REVIEW

File: views/match/match_performances.jade [hunk 1]
```diff
@@ -12,6 +12,7 @@ block match_content
               th: abbr(title=tooltips.lane_efficiency) EFF@10
               th: abbr(title=tooltips.lhten) LH@10
               th: abbr(title=tooltips.dnten) DN@10
+              th Stacked
```
Comment (howardchung) [inline views/match/match_performances.jade]: add a tooltip

File: views/match/match_performances.jade [hunk 2]
```diff
@@ -21,6 +22,7 @@ block match_content
             td.rankable #{(Number(player.lane_efficiency)*100).toFixed(2)}
             td.rankable #{player.lh_t && player.lh_t[10] ? player.lh_t[10] : "-"}
             td.rankable #{player.dn_t && player.dn_t[10] ? player.dn_t[10] : "-"}
+            td #{player.creeps_stacked || "-"}
```
Comment (howardchung) [inline views/match/match_performances.jade]: rankable

File: views/match/match_performances.jade [hunk 3]
```diff
@@ -12,6 +12,7 @@ block match_content
               th: abbr(title=tooltips.lane_efficiency) EFF@10
               th: abbr(title=tooltips.lhten) LH@10
               th: abbr(title=tooltips.dnten) DN@10
+              th: abbr(title=tooltips.creeps_stacked) Stacks
```
Comment (howardchung) [inline views/match/match_performances.jade]: change tooltip name

**Comment by diw1** [comment]
cannot wait the get the stack stats for fantasy challenge!

**Comment by albertcui** [comment]
LGTM. OK to deploy.

**Pull request merged**